### ISSUE TITLE
NVDA does not read categories when inserting Icons in Office 365

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#10595]
### Is your feature request related to a problem? Please describe.

In Word 365:
1. In a blank document, press alt+n, n, s to select "Icons" from the Insert ribbon.
2. Press TAB twice to move from the edit, past the categories, to the icon list itself.
3. NVDA reports the movement
4. Press the down arrow twice to move past the first (Accessibility) category, to the first item in the Analytics category.

Actual behaviour:
At step 3, NVDA reads a handful of icons / categories before reading "Wheelchair access, not selected" (the first icon).
At step 4, NVDA reads "List" before reading the item.  NVDA reports list each time a new category is reached but does not report which list.  

The layout of this dialog is the edit box where you can type something to search for (Braille, or Train or dog for instance), a list of categories which will let you jump directly to that category, and on the right the list of icons which lists all the icons from one category, then all the icons from the next category and so on - there is a category heading in between each section.  I found I can get to this with object navigation by moving to the parent object, then back one object.

I couldn't get this list to read with Narrator, however, Microsoft advises it works in a more recent version.  Further to this, they advise "Narrator is announcing those category names as a part of a feature known as Context Awareness.  It uses UIA relationships and such to determine when an object’s parent information has changed."

### Describe the solution you'd like

NVDA to read icon categories when noting the new "list".

### Additional context
Windows 10 Version: 
18362.388 
Office 365 Version: 
16.0.12228.20322

**Comment by Adriani90** [comment]
This works as expected now with NVDA 2024.2 in Office 365 Microsoft 365 MSO (Version 2405 Build 16.0.17628.20006) 64 Bit or newer.
Closing as works for me.
